book/01-introduction/sections/help.asc

```diff
@@ -1,46 +1,46 @@
-[[_git_help]]
-=== Getting Help
+[[_گرفتن_کمک]]
+=== کمک گرفتن
 
-If you ever need help while using Git, there are two equivalent ways to get the comprehensive manual page (manpage) help for any of the Git commands:
+اگر در هنگام استفاده از گیت همیشه به کمک نیاز دارید، برای دستیابی به راهنمای دستی جامع (صفحه من)هر یک از دستورات گیت دو روش معادل وجود دارد :
 
-[source,console]
+[منبع، کنسول]
 ----
 $ git help <verb>
 $ man git-<verb>
 ----
 
-For example, you can get the manpage help for the `git config` command by running(((git commands, help)))
+به عنوان مثال، برای اجرای دستور پیکربندی گیت، شما می توانید به راهنمای دستی جامع با اجرا کردن آن دسترسی پیدا کنید.(((گیت دستورات, کمک)))
 
-[source,console]
+[منبع، کنسول]
 ----
-$ git help config
+$ گیت پیکربندی کمک
```

Comment:
This is a command line! Have you tried using Git before starting the translation? You should have used Git before trying to translate the book.